routing/unified_policies.go

```diff
@@ -258,7 +279,7 @@ func (u *unifiedPolicy) getPolicyNetwork(
 	modifiedPolicy := *bestPolicy
 	modifiedPolicy.TimeLockDelta = maxTimelock
 
-	return &modifiedPolicy
+	return &modifiedPolicy, maxCapMsat.ToSatoshis()
```

Comment:
Why not stick to msat to avoid rounding max htlc?

Reply:
As I'm now using `unifiedPolicyEdge` we would have to change it there as well to transport the new units. Also the rounding seems to be not very impactful here